lib/Backends/Interpreter/Interpreter.cpp

```diff
@@ -63,6 +63,7 @@ bool Interpreter::isOpSupported(Kinded::Kind opKind, ElemKind elementTy) const {
     case Kinded::Kind::SelectNodeKind:
     case Kinded::Kind::SigmoidNodeKind:
     case Kinded::Kind::SliceNodeKind:
+    case Kinded::Kind::SparseLengthsWeightedSumNodeKind:
```

Comment:
do you plan to add support for CPU as well?

Reply:
Yes, probably quite soon.